### DISCUSSION TITLE
Query regarding the removal of less secure apps access from google account

### DISCUSSION THREAD

**Discussion by AnkitThakker47** [saran-surya/email_auth#66] [Q&A] [answered]
I recently just created a new Gmail account for using it for the server for sending emails. While creating the email and enabling the less secure app access I saw that the less secure app access setting will be removed from 30th May 2022. So I had a doubt that will I still be able to send OTP emails if the setting is removed since it is a necessary step for setting up the Gmail account.

<img width="634" alt="image" src="https://user-images.githubusercontent.com/50229717/164749657-82815f51-8dae-4223-bb82-c9d96bcefaca.png">

Also, it is written that it will be turned off if not being used and also written that it will be removed permanently from 30th May 2022. So just wanted a clarification on this. Other than that everything is working fine.

**Comment by saran-surya** [comment]
Hi @AnkitThakker47 , 
I will look into this issue, And will update you regarding it. We can try some other process as a work around. Hoping for the best.

Update : 26/4/2022
 - Will look into options of implementing OAuth / more secure apps to the test servers on the first phase, And the plugin will be updated for the remote servers

**Reply by rabukakeher** [reply]
Hello  @saran-surya,
Is there any update on this? looks like it really isn't working anymore. Really appreciate your work on this project.

Thanks,
RB

**Comment by saran-surya** [comment]
Hi, currently I am occupied with other works, I am also finding time to
implement the oauth and other steps, kindly expect a delay.


Thankyou for your time

**Comment by saran-surya** [comment, marked as answer]
A Patch to the test servers were made a bit ago, But with google completely removing the Less secure apps, usage of OUTLOOK seemed to be a better alternative, thus configuring the plugin to use outlook as a default setting, a new patch for GOOGLE OAuth will be created, that includes an automation script to create the cloud servers, Or a detailed documentation will be made.

Thankyou
